### PR TITLE
fix: client app view layout

### PR DESCRIPTION
**Pull request by antlio**
this pr fixes client app views layout after moving address bar to the request view.

### CODE REVIEW

**Review by cameronrohani — APPROVED** [review]

File: packages/client-app/src/components/ViewLayout/ViewLayout.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
-  <div class="flex xl:flex-row flex-col xl:gap-0 gap-1 custom-scroll rounded">
+  <div
+    class="m-1 mt-0 flex min-h-0 flex-1 rounded xl:rounded-lg xl:border xl:overflow-hidden leading-[19.5px]">
```
Comment (amritk) [inline packages/client-app/src/components/ViewLayout/ViewLayout.vue:3]: 😢

Comment (antlio) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue:3]: i just moved what was already there, leading?

Comment (marclave) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue:3]: i think its the pixel values 👀 maybe we can add a linter or smth

Comment (amritk) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue:3]: yes I die a little inside every time

Comment (amritk) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue:3]: yep there's a lint rule! Will add

https://www.npmjs.com/package/eslint-plugin-tailwindcss

Comment (antlio) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue:3]: let me check that out real quick

Comment (amritk) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue]: ❤️

Comment (antlio) [reply packages/client-app/src/components/ViewLayout/ViewLayout.vue]: updated. note to myself our variable are messing with default tw leading sizes, to be checked

**Review by marclave — APPROVED** [review]
🫡

**Pull request merged**